README.md

```diff
@@ -283,8 +283,7 @@ Using `--lock-ttl=30m` will allow other nodes to take over if TTL has expired (i
 
 ## Building
 
-See the [CircleCI config](.circleci/config.yml) for the preferred
-version of Golang. Kured now uses [Go
+Kured now uses [Go
```

Comment:
Maybe point to `go.mod` instead now?

Reply:
Yes, I could do that in a follow up patch, or in this one.